### PR TITLE
Add app icons

### LINKED ISSUE #6: Add App Icons

### PR DESCRIPTION
**Pull request by cparrapa**
closes #6

### CODE REVIEW

**Comment by cparrapa** [comment]
Hi @chico i just added the main buttons to this branch, so that you can use to the play screen #8 , please check and let me know if ready to merge

File: App/Themes/Images.js [hunk 1]
```diff
@@ -9,5 +9,8 @@ export default {
   },
   joystick: {
     arrows: require('App/Images/Joystick/arrows.png')
+  },
+  buttons: {
+    border: require('App/Images/Buttons/Border/border.png')
   }
```
Comment (chico) [inline App/Themes/Images.js]: @cparrapa new buttons look great, can you add them to `App/Themes/Images.js`?

**Comment by cparrapa** [comment]
@chico is here https://github.com/OttoDIY/OttoDIYApp/tree/6-add-app-icons/App/Images/Buttons/Plus but maybe we should create a folder for the robots icons?

**Comment by chico** [comment]
@cparrapa yes, I think it's best to create a new folder called `Robots` and put the plus images there.

**Pull request merged**